Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -548,7 +548,6 @@ namespace Multiplayer
             m_renderBlendFactor = 0.0f;
             m_lastReplicatedHostTimeMs = packet.GetHostTimeMs();
             m_lastReplicatedHostFrameId = packet.GetHostFrameId();
-            m_networkTime.AlterTime(m_lastReplicatedHostFrameId, m_lastReplicatedHostTimeMs, AzNetworking::InvalidConnectionId);
```

Comment:
If you remove this, then the time on the local client will no longer reflect the time replicated by the server. I’d have to double check but I believe this will break rewind.

Reply:
This logic was never firing due to high m_lastReplicatedHostFrameId was defaulted to and I'm seeing correct rewind on the player so that doesn't seem to be the case. This actually asserted non-stop on regular updates since it set time too far back.

Reply:
That all being said, maybe this hints at a bigger issue in time management yet to be diagnosed.

Reply:
Yup, this definitely requires a deeper dive.. as the time on the client will never advance with this change.

Reply:
Okay this seems fine in Profile for some reason so I'm restoring it for now. More investigation required in debug however.